File: backend/moyeo/src/main/java/com/mo/moyeo/domain/bank/entity/Bank.java
package com.mo.moyeo.domain.bank.entity;

import jakarta.persistence.*;
import lombok.*;

@Getter
@Entity
@Table(name = "banks")
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Builder
public class Bank {

    @Id
    @Column(name = "bank_code", length = 3)
    private String code;

    @Column(name = "bank_name", length = 50, nullable = false)
    private String bankName;

    @Column(name = "logo_img")
    private String logoImg;
}

File: frontend/app/src/main/java/com/d108/moyeo/domain/model/Bank.kt
package com.d108.moyeo.domain.model

import androidx.compose.ui.graphics.ImageBitmap

data class Bank(  // 서버에서 받은 은행 정보를 UI에서 활용하기 좋게 변경
    val code: String,
    val name: String,
    val logoBitmap: ImageBitmap? //디코딩된 이미지를 담을 속성 추가 (실패할 수 있으므로 Nullable)
)
